server/utils/reviewNotifier.js

```diff
@@ -1,22 +1,21 @@
  import dotenv from 'dotenv';
  import mailer from './mailer';
- import reviewNotifierTemplate from './EmailTemplates/reviewNotifierTemplate';
+ import reviewNotifierTemplate from './emailTemplates/reviewNotifierTemplate';
 
  dotenv.config();
 
  /**
   * @description function for sending notification email when a user's recipe gets a review
   * @function
   *
-  * @param   {object} modelR      Recipes model
-  * @param   {object} modelU      Users model
-  * @param   {number} recipeId    ID of the recipe been reviewed
-  * @param   {string} reviewBody  Content of the review
-  * @param   {string} reviewer    The username of the person who made the review
+  * @param   {object} modelR       - Recipes model
+  * @param   {object} modelU       - Users model
+  * @param   {number} recipeId     - ID of the recipe been reviewed
+  * @param   {object} postedReview - Object content of the review post
   *
+  *@returns  {function}            - The nodemailer function that sends mail the user
   */
-
- const reviewNotifier = (modelR, modelU, recipeId, reviewBody, reviewer) => {
+ const reviewNotifier = (modelR, modelU, recipeId, postedReview) => {
```

Comment:
Rename the arguments for this function, modelU and modelR. modelR could be recipesModel and modelU could be usersModel